src/plugins/intel_gna/src/layers/gna_layer_info.hpp

```diff
@@ -62,15 +62,15 @@ class LayerInfo {
     // The name of the funciton may be somehwat misleading
     // Explanation: when in low precision mode the listed layers have 8-bit outputs
     // and when in 16-bit input mode, they have 16-bit outputs
-    bool has8BOr16BOutput() const noexcept {
+    bool has8BOr16BOutput() const {
         IS_VALID();
         static InferenceEngine::details::caseless_set<std::string> layersWith8BOr16BOutputs = {"memory", "input", "split", "slice", "concat", "copy", "const"};
         return layersWith8BOr16BOutputs.find(layer->type) != layersWith8BOr16BOutputs.end() ||
                isActivation() ||
                (isCrop() && !isCropAffined()) ||
                isPermute();
     }
-    bool has32BOutput() const noexcept {
```

Comment:
What function inside this one throws an exception?

Reply:
isCropAffined() use GetParams which my throw

Reply:
nice catch